packages/react/src/components/Popup/Popup.test.tsx

```diff
@@ -0,0 +1,54 @@
+import * as React from 'react';
+import { Popup } from './Popup';
+import { isConformant } from '../../common/isConformant';
+import { render } from '@testing-library/react';
+import { resetIds } from '../../Utilities';
+
+describe('Popup', () => {
+  afterAll(() => {
+    resetIds();
+  });
+
+  isConformant({
+    Component: Popup,
+    displayName: 'Popup',
+  });
+
+  it('defaults to enableAriaHiddenSiblings=true', () => {
+    const { getByText } = render(
+      <div>
+        <div>sibling</div>
+        <Popup>content</Popup>
+      </div>,
+    );
+
+    const bodyChildren = Array.from(document.body.children) as HTMLElement[];
+
+    const content = getByText('content');
+    const contentParent = bodyChildren.find(el => el.contains(content));
+    expect(contentParent).toBeTruthy();
+    expect(contentParent!.getAttribute('aria-hidden')).toBeNull();
+
+    for (const node of bodyChildren) {
+      if (node !== contentParent) {
+        expect(node.getAttribute('aria-hidden')).toBe('true');
```

Comment:
Same issue here as in the Dialog tests (and Modal tests, it looks like). I did double check this one, and it looks like this line is never reached, and the `expect` doesn't get called.